charts/internal/calico-monitoring/templates/scrapeconfig-bird.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: ScrapeConfig
+metadata:
+  name: shoot-calico-bird
+  namespace: {{ .Release.Namespace }}
+  labels:
+    prometheus: shoot
+spec:
+  authorization:
+    credentials:
+      name: shoot-access-prometheus-shoot
+      key: token
+  scheme: HTTPS
+  tlsConfig:
+    # This is needed because the kubelets' certificates are not are generated
+    # for a specific pod IP
+    insecureSkipVerify: true
+  kubernetesSDConfigs:
+  - apiServer: https://kube-apiserver
+    authorization:
+      credentials:
+        name: shoot-access-prometheus-shoot
+        key: token
+    followRedirects: true
+    namespaces:
+      names:
+      - kube-system
+    role: Endpoints
+    tlsConfig:
+      # This is needed because we do not fetch the correct cluster CA bundle right now
+      insecureSkipVerify: true
+  metricRelabelings:
+  - sourceLabels:
+    - __name__
+    action: keep
+    regex: bird_.+
```

Comment:
This retains all metrics that aren't the "meta" metrics. Do you need all of them?

Reply:
We are mostly interested in `bird_protocol_up` to monitor BGP session state. 

Additionally, `bird_protocol_prefix_.` metrics might be interesting. Other protocols like OSPF are not relevant for for Calico CNI use case. We could narrow them to above and extend if required.

Reply:
Makes sense, we did evaluate and did reduce them. Basically we need eventually only `bird_protocol_up` for our alerts, but will be also keeping `bird_protocol_uptime`, `bird_protocol_prefix_export_count` and `bird_protocol_prefix_export_count`.